shell/assets/translations/en-us.yaml

```diff
@@ -7718,7 +7718,10 @@ performance:
     label: Server-side Pagination
     description: By default Lists will fetch all resources and paginate in the browser (create the page given sorting and filtering settings). Change this setting to enable pagination server-side. This should help the responsiveness of the UI in systems with a lot of resources.
     checkboxLabel: Enable Server-side Pagination
-    applicable: "This applies to the following resource types"
+    checkboxUseDefault:
+      label: Enable Custom Resources
+      placeholder: Customize the resources that server-side pagination applies to. This is an advanced feature and must be managed outside of the UI. Customized resource types are not updated on Rancher upgrade and must be manually changed.
```

Comment:
I would think we'd like to link to some docs so the user can do this if they so choose.

Reply:
Good comment!

We're on a big push for a cleaner UI and stuff like this is pretty advanced, we wouldn't expect users to exercise it without direct instruction / instructions from us anyway. With all that in mind just nuked it.